workflows/dags/data/bulk_data_harvest.py

```diff
@@ -143,6 +146,20 @@ def add_keywords(record, builder):
                     for value in item:
                         builder.add_keyword(value)
 
+        def add_date(record, builder):
+            """Add date to the record."""
+            creation_date = record["creation_date"]
+            last_updated = record.get("last_updated", None)
```

Comment:
```suggestion
            last_updated = record.get("last_updated")
```

by default it's None, no need to specify